Sync-Introduction.md

```diff
@@ -1,23 +1,17 @@
 ---
 uid: Sync-Introduction
-title: Sync Introduction
+title: Downloads & Sync
 legacyUrl: /support/solutions/articles/44001161893-sync-introduction
 ---
 
-With Emby Sync, you can choose specific media, then a device or destination such as a cloud provider along with criteria such as resolution and bit rate and Emby will prepare/convert the media to the format requested, then copy it to the destination.
+The Downloads and Sync feature is an [Emby Premiere](Emby-Premiere.md) feature offering users and server admin, the ability to copy media at specific quality resolution, to devices for offline streaming or to folders to provide backup and converted versions of the media. The latter requires the installation of the **Folder Sync** plugin.
```

Comment:
at specific quality resolution...

=> at specific resolutions and bit rates to devices for offline playback or to server folders for streaming alternate versions. The latter requires the installation of the **Folder Sync** plugin.

Reply:
Replaced paragraph with

`The Downloads and Sync feature is an [Emby Premiere](Emby-Premiere.md) feature offering users and server admins the ability to copy media at specific resolutions and bit rates to devices for offline playback or to server folders for backup or streaming alternate versions. The use of server folders requires the installation of the [Folder Sync](Folder-Sync.md) plugin.`